cipher/cipher.py

```diff
@@ -5,7 +5,6 @@ class Cipher(ABC):
     """
     Base class for cryptocryphy
     """
-    @abstractmethod
```

Comment:
why do you need this?

Reply:
I remove the abstractmethod decorator from __init__ since it seems if the method has been assigned the abstractmethod decorator then the class won't be initialized.